src/index.tsx

```diff
@@ -481,39 +474,26 @@ export class Modalize<FlatListItem = any, SectionListItem = any> extends React.C
   private onKeyboardShow = (event: any) => {
     const { height } = event.endCoordinates;
 
-    this.setState({ keyboardToggle: true });
-    this.onContentViewChange(height);
+    this.setState({ keyboardToggle: true, keyboardHeight: height });
   };
 
   private onKeyboardHide = () => {
-    this.setState({ keyboardToggle: false });
-    this.onContentViewChange();
+    this.setState({ keyboardToggle: false, keyboardHeight: 0 });
   };
 
-  private renderComponent = (Tag: React.ReactNode, name: string): React.ReactNode => {
+  private renderComponent = (Tag: React.ReactNode): React.ReactNode => {
     // @ts-ignore
     const element = React.isValidElement(Tag) ? Tag : <Tag />;
 
-    // We don't need to calculate header and footer if they are absolutely positioned
-    if (Tag && hasAbsoluteStyle(Tag)) {
-      return element;
-    }
-
-    const onLayout = ({ nativeEvent }: LayoutChangeEvent) =>
-      this.setState(
-        { [`${name}Height`]: nativeEvent.layout.height } as any,
-        this.onContentViewChange,
-      );
-
     return (
-      <View style={s.component} onLayout={onLayout} pointerEvents="box-none">
+      <View style={s.component} pointerEvents="box-none">
```

Comment:
I would say that the View is not necessary here anymore since you removed the onLayout props

Reply:
I agree, but I was afraid to clean up, I'll try